solana/solana-typegen/src/program/anchor/old.spec.ts

```diff
@@ -360,7 +360,10 @@ function fromType(type: IdlType): Type {
             type: fromType(type.option),
         }
     } else if ('coption' in type) {
-        throw new Error(`Type "coption" is not supported`)
+        return {
+            kind: TypeKind.Option,
```

Comment:
afaik `COption` uses `u32` as flag, so it's not compatible with borsch `option`

Reply:
I tested with token program and was able to decode coption with pubkey, u32 and u8 with option. I think  they have the same codec

Reply:
@dariaag no they don't
https://github.com/solana-labs/solana-program-library/blob/token-client-v0.10.0/token/program/src/state.rs#L255

Reply:
ok. maybe we should add it as a separate type, since solana system programs use it? we need it if we want to be able to index spl20 token transfers.